central/notifier/service/service_impl_test.go

```diff
@@ -19,17 +21,18 @@ import (
 )
 
 func TestNotifierService(t *testing.T) {
-	t.Parallel()
+	t.Setenv("ROX_SECURED_CLUSTER_NOTIFICATIONS", "true")
 	suite.Run(t, new(notifierServiceTestSuite))
```

Comment:
I thought about that, and then since the path is the same except for the actual send decided against. I can see what I can do without repeating the test code.